bridge_adaptivity/config/settings/base.py

```diff
@@ -128,6 +128,17 @@
     },
 }
 
+# Celery settings
+
+# Timespan for running sync task in seconds
+CELERY_DELAY_SYNC_TASK = 300  # default value is equal to 5 minutes
```

Comment:
I would be more clear to write like this 
```CELERY_DELAY_SYNC_TASK = 5 * 60  # default value is equal to 5 minutes```

Reply:
Thanks, that is a nice catch. Improved.